generate/beat/{beat}/NOTICE

```diff
@@ -0,0 +1,5 @@
+{beat}
+Copyright 2017 {full_name}
```

Comment:
The year is generated by the script, so you would need to re-run the script every year, at least :)